x-pack/plugins/event_log/server/es/init.ts

```diff
@@ -230,14 +235,10 @@ class EsInitializationSteps {
       this.esContext.esNames.dataStream
     );
     if (!exists) {
-      await this.esContext.esAdapter.createDataStream(this.esContext.esNames.dataStream, {
-        aliases: {
-          [this.esContext.esNames.dataStream]: {
-            is_write_index: true,
-            is_hidden: true,
-          },
-        },
-      });
+      await this.esContext.esAdapter.createDataStream(this.esContext.esNames.dataStream);
+    } else {
+      // apply current mappings to existing data stream
+      await this.esContext.esAdapter.updateConcreteIndices(this.esContext.esNames.dataStream);
```

Comment:
Do we end up testing this path in FTR?  I'm thinking no, but not positive.

Could we do this with a jest integration test?  Just start a Kibana, then start another or kill the first and restart.  Maybe we could add a debug log to ensure we made it here?

Reply:
Added a debug log when the `PUT` mappings call is successful and added a jest integration test that starts up Kibana, tests for the `Creating datastream` info log, then restarts Kibana and tests for the `Updating concrete indices` info log and the success debug log